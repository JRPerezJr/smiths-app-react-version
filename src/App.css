@import url('https://fonts.googleapis.com/css?family=Spartan:400,700&display=swap');

:root {
  --background: #c9ced3;
  --on-background: rgb(66, 66, 66);
  --player-background: #e7e7e7;
  --card-shadow: 0 15px 30px 5px rgba(0, 0, 0, 0.3);
  --image-shadow: 0 5px 30px 5px rgba(0, 0, 0, 0.5);
  --progress-container: #fff;
  --progress-bar: #242323;
  --player-controls: rgb(129, 129, 129);
}

[data-theme='dark'] {
  --primary-color: #ffffff;
  --background: #000000;
  --on-background: #ffffff;
  --player-background: #1e1e20;
  --progress-container: #ffffff;
  --progress-bar: #45d4f7;
  --player-controls: #ffffff;
  --title: #06bcfa;
  --bmw-cluster-dark: #05a3e6;
}

html {
  box-sizing: border-box;
}

body {
  margin: 0;
  min-height: 100vh;
  background: var(--background);
  display: flex;
  justify-content: center;
  align-items: center;
  font-family: Spartan, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
    Helvetica, Arial, sans-serif;
  font-size: 12px;
}
