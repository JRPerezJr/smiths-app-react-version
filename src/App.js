import React from 'react';
import './App.css';

function App() {
  return (
    <div className="App">
      <h1>Welcome to The Smiths Player</h1>
    </div>
  );
}

export default App;
