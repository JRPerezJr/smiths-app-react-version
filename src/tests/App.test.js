import React from 'react';

import renderer from 'react-test-renderer';

// import { mount } from 'enzyme';

import App from '../App';

describe('App component tests', () => {
  it('Sanity test for App component', () => {
    console.log('App component sanity test passed!');
  });

  test('snapshot renders without errors', () => {
    const component = renderer.create(<App />);
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });

  // test('renders main title', () => {

  // });
});
