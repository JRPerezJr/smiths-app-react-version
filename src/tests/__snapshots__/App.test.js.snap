// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component tests snapshot renders without errors 1`] = `
<div
  className="App"
>
  <h1>
    Welcome to The Smiths Player
  </h1>
</div>
`;
